packages/forms/frontend/sirius-components-forms/src/propertysections/__tests__/CheckboxPropertySection.test.tsx

```diff
@@ -65,6 +66,7 @@ const checkboxWithEmptyStyle: GQLCheckbox = {
   booleanValue: false,
   style: {
     color: '',
+    labelPlacement: null,
```

Comment:
labelPlacement can't be null right?

Reply:
In fact, this test is not realistic on this point, I changed it to use the default value.